packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -559,7 +567,8 @@ export class Relayer {
   private async _sendTransaction(
     transaction: TransactionType,
     message: string,
-    mrkdwn: string
+    mrkdwn: string,
+    level = "info"
```

Comment:
level defaults to `info` (what it was before). else, you can define a `level` that will be set on message emmited. we simply append "error" onto the `level` on a transaction.